clang/lib/Headers/hlsl/hlsl_detail.h

```diff
@@ -33,6 +33,14 @@ constexpr enable_if_t<sizeof(U) == sizeof(T), U> bit_cast(T F) {
   return __builtin_bit_cast(U, F);
 }
 
+constexpr vector<uint, 4> d3d_color_to_ubyte4(vector<float, 4> V) {
```

Comment:
```suggestion
constexpr vector<uint, 4> D3DColorToUByte4Impl(vector<float, 4> V) {
```
I think this follows more to how the length builtin was defined